frontend/css/style.css

```diff
@@ -272,3 +272,85 @@ progress::-webkit-progress-bar {
     height: 100%;
     border-radius: 1.5em;
 }
+
+/** Subjects on the homepage **/
```

Comment:
subject container styles are duplicate, may we plz remove them from line 276 to 307

Reply:
I am gonna remove the styles you nested inside .main